cerk_port_mqtt/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cerk_port_mqtt"
-version = "0.2.6"
+version = "0.3.0"
```

Comment:
this will be done in a separate release process

https://github.com/ce-rust/cerk#release-management